.github/workflows/unit-test.yml

```diff
@@ -20,20 +20,13 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
-      - name: Golangci-lint
-        uses: golangci/golangci-lint-action@v2
-        with:
-          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.32
-          args: --timeout 2m
-
       - name: Get kubebuilder
         env:
-          version: 3.1.0 # latest stable version
+          version: 2.3.1 # latest stable version
```

Comment:
Revert this change ?

Reply:
3.1.0 version is not working. will have to check and fix in next pr.

Reply:
3.1.0 version is not working. will have to check and fix in next pr.